assets/mods/generous-rebels/data/scripts/main.lua

```diff
@@ -1,9 +1,10 @@
 JA2Require("enums.lua")
+RegisterListener("BeforePrepareSector", "mod:generous-rebels", "addItemsToSector")
```

Comment:
This is odd, why do we hardcode mod names?

Reply:
My thinking is that, there may eventually be multiple mods listening to this same Observable. So that we need a unique listener key to prevent them from overriding each other.

Reply:
This partly defeats the point of externalization. Can we not use random keys or let the mods specify their namespace somewhere?

Reply:
A mod may needs to:
- Disable vanilla listeners
- Register multiple listeners with the same Observables 

So yes, perhaps we can make vanilla listeners have well-known names, then the engine decides the random and/or namespaced keys for the mods'.